tests/functional/z_cluster/rag/conftest.py

```
@@ -0,0 +1,46 @@
+"""
+import logging
+import pytest
+
+from ocs_ci.helpers.ols_helpers import cleanup_ols_operator, do_deploy_ols
+
+log = logging.getLogger(__name__)
+
+
+
+@pytest.fixture(scope="class")
+def ols_cleanup(request):
```

Ruff (0.15.15)

[warning] Unused function argument: `request`

(ARG001)